applications/ecallmgr/src/ecallmgr_call_events.erl

```diff
@@ -637,6 +638,16 @@ generic_call_event_props(Props) ->
      ,{<<"To-Tag">>, props:get_value(<<"variable_sip_to_tag">>, Props)}
      ,{<<"Switch-URL">>, props:get_value(<<"Switch-URL">>, Props)}
      ,{<<"Switch-URI">>, props:get_value(<<"Switch-URI">>, Props)}
+     ,{<<"Switch-Nodename">>, props:get_value(<<"Switch-Nodename">>, Props)}
+     ,{<<"Channel-State">>, get_channel_state(Props)}
+     ,{<<"Channel-Call-State">>, props:get_value(<<"Channel-Call-State">>, Props)}
+     ,{<<"Channel-Name">>, props:get_value(<<"Channel-Name">>, Props)}
+     ,{<<"Channel-Name">>, props:get_value(<<"Channel-Name">>, Props)}
+     ,{<<"Channel-Is-Loopback">>, get_is_loopback(props:get_value(<<"variable_is_loopback">>, Props))}
+     ,{<<"Channel-Loopback-Leg">>, props:get_value(<<"variable_loopback_leg">>, Props)}
+     ,{<<"Channel-Loopback-OtherLeg-ID">>, props:get_value(<<"variable_other_loopback_leg_uuid">>, Props)}
```

Comment:
@lazedo Our other events use "*-Other-Leg-*", I would prefer to keep that consistent:

     ,{<<"Other-Leg-Direction">>, props:get_value(<<"Other-Leg-Direction">>, Props)}
     ,{<<"Other-Leg-Caller-ID-Name">>, props:get_value(<<"Other-Leg-Caller-ID-Name">>, Props)}
     ,{<<"Other-Leg-Caller-ID-Number">>, props:get_value(<<"Other-Leg-Caller-ID-Number">>, Props)}
     ,{<<"Other-Leg-Destination-Number">>, props:get_value(<<"Other-Leg-Destination-Number">>, Props)}
     ,{<<"Other-Leg-Call-ID">>, get_other_leg(Props)}